src/SpaceTime/2d/Acceleration2.cs

```diff
@@ -0,0 +1,272 @@
+﻿using System;
+using Bearded.Utilities.Math;
+using OpenTK;
+
+namespace Bearded.Utilities.SpaceTime
+{
+    /// <summary>
+    /// A type-safe representation of a 2d directed acceleration vector.
+    /// </summary>
+    public struct Acceleration2 : IBackedBy<Vector2>, IEquatable<Acceleration2>
+    {
+        private readonly Vector2 value;
+
+        #region construction
+
+        /// <summary>
+        /// Creates a new instance of the Acceleration2 type.
+        /// </summary>
+        public Acceleration2(Vector2 value)
+        {
+            this.value = value;
+        }
```

Comment:
It appears to be the style of the rest of the library. Overloads I can see, but sometimes the grouping is a bit more subjective. I live separating the methods, I've never been one for the crammed look. It's up to you, vertical whitespace is always a bit of a weird thing, so I don't think it makes sense to put solid rules on that.